sugargame/event.py

```diff
@@ -173,7 +173,7 @@ def _keyevent(self, widget, event, type):
             ukey = chr(Gdk.keyval_to_unicode(event.keyval))
             if ukey == '\000':
                 ukey = ''
-            evt = pygame.event.Event(type, key=keycode, unicode=ukey, mod=mod)
+            evt = pygame.event.Event(type, key=keycode, str=ukey, mod=mod)
```

Comment:
Why change the name of this argument?  It looks like an accident.  Also, please synchronise the sugargame files with the sugargame repository rather than port the files in this repository.